homeassistant/components/deconz/cover.py

```diff
@@ -66,12 +63,12 @@ def __init__(self, device, gateway):
     @property
     def current_cover_position(self):
         """Return the current position of the cover."""
-        return 100 - int(self._device.brightness / 254 * 100)
+        return 100 - self._device.position
 
     @property
     def is_closed(self):
         """Return if the cover is closed."""
-        return self._device.state
+        return self._device.is_open is False
```

Comment:
Can it be a non-boolean value or would this work:
```suggestion
        return not self._device.is_open
```

Reply:
absolutely